scripts/comparator/queries.json

```diff
@@ -80,5 +80,16 @@
       "30s", 
       "1m"
     ]
+  },
+  {
+    "queryGroup":"subfunction",
+    "queries":[
+      "avg_over_time(sum_over_time(quail[1m])[5m:5m])",
+      "avg_over_time(rate(quail[1m])[10m:5m])",
+      "avg_over_time(sum_over_time(quail[1m])[10m:5m])"
+    ],
+    "steps" : [
+      "5m"
+    ]
```

Comment:
Should we add some negatively failing cases that are commented out so we know what to tackle for future work?

Reply:
JSON unfortunately :p 

But will add ability for that